app/models/patient.rb

```diff
@@ -126,6 +111,10 @@ def remaining_login_attempts
     MAX_LOGIN_ATTEMPTS - login_attempts
   end
 
+  def age
+    @age ||= ((Time.zone.now - birthday.to_time) / 1.year.seconds).floor # rubocop:disable Rails/Date
```

Comment:
Entendi nada, hahaha. Pode explicar por que a necessidade dessa aritmética?

Reply:
Para calcular a idade como humanos contam (arredondando para baixo) é isso mesmo :-) Eu desconheço uma versão melhor, fico feliz em mudar.

Reply:
Saquei, é que ficou bem baixo nível e tive dificuldade de entender. O que dificultou um pouco é saber o resultado do `Time.zone.now - birthday.to_time`. Sugestão pra isso ficar mais claro:

```suggestion
    time_from_birth_in_seconds = Time.zone.now - birthday.to_time
    @age ||= (time_from_birth_in_seconds / 1.year.seconds).floor # rubocop:disable Rails/Date
```